docs/cases/cases-overview.asciidoc

```diff
@@ -118,9 +118,7 @@ IMPORTANT: Before importing Lens visualizations, Timelines, or alerts into a spa
 [float]
 [[cases-export]]
 === Export a case
-Use the export option to move cases between different Kibana instances. When you export a case, the following data is exported to a newline-delimited JSON (NDJSON) file: case details, user actions, text string comments, case alerts, and lens visualizations.
-
-NOTE: Exported alerts contain metadata that allow the case to reference the appropriate alerts after they're imported. Lens visualizations are exported as JSON blobs.
+Use the export option to move cases between different Kibana instances. When you export a case, the following data is exported to a newline-delimited JSON (`.ndjson`) file: case details, user actions, text string comments, case alerts, and lens visualizations (which are exported as JSON blobs).
```

Comment:
Should "export" be in quotes or bold if it's an option in the UI?

Reply:
Yep, the button is actually named Export so I'll bold it here.